src/rust/engine/fs/src/glob_matching.rs

```diff
@@ -536,6 +536,7 @@ trait GlobMatchingImplementation<E: Display + Send + Sync + 'static>: VFS<E> {
       .into_inner()
       .into_iter()
       .collect::<Vec<_>>();
+    #[allow(clippy::unnecessary_sort_by)]
```

Comment:
What's the deal with these? Is there a bug for false positives we can link to?

Reply:
Nothing that I see in the [Clippy issues tracker](https://github.com/rust-lang/rust-clippy/issues). I am opening an issue there now.

Reply:
https://github.com/rust-lang/rust-clippy/issues/5976